app/scripts/metamask-controller.js

```diff
@@ -1521,6 +1521,15 @@ export default class MetamaskController extends EventEmitter {
       ),
       getTransactions: txController.getTransactions.bind(txController),
 
+      // update transaction methods
```

Comment:
I think we can remove this since the method names explain this and to keep the methods together under the comment above
```
// txController
``` 


```suggestion
```

Reply:
Fixed [here](https://github.com/MetaMask/metamask-extension/pull/13641/commits/b96851034b9e1b43000aac2c4db71efca309ec08)